src/main/java/org/springframework/data/couchbase/core/query/N1QLQuery.java

```diff
@@ -45,4 +47,21 @@ public JsonObject n1ql() {
 		return query;
 	}
 
+	@Override
+	public String toN1qlSelectString(ReactiveCouchbaseTemplate template, String collectionName, Class domainClass,
+			Class returnClass, boolean isCount, String[] distinctFields) {
+		/*
```

Comment:
this big block commented out is odd - why?

Reply:
cut-and-paste but not used.
Fixed.